VIDEO_4_SCRIPT.md

```diff
@@ -0,0 +1,47 @@
+# Video 4: AsyncAPI and OpenAPI
+
+Hello again! Welcome to a new video about AsyncAPI. This video will discuss what AsyncAPI and OpenAPI have in common and how they differ. 
+
+Ready or not, here we go! 
+
+## What is OpenAPI?
+
+OpenAPI is a standard for describing application programming interfaces (APIs). This specification defines an open, vendor-independent description format for API services. In particular, OpenAPI can describe, develop, test, and document REST-compliant APIs.
+
+The current OpenAPI specification grew out of the predecessor project, Swagger. The development company SmartBear subjected the existing Swagger specification to an open license, handing over both maintenance and development of the initiative to OpenAPI. In addition to SmartBear, members of the OpenAPI initiative include industry giants such as Google, IBM, and Microsoft. The Linux Foundation also supports this project. 
+
```

Comment:
I believe it will help to briefly discuss the style(REST, message-driven) employed by these two API types so it wouldn't seem strange when a reference is made to them later in the script.
not sure I'm technically correct to refer to them as style though.

Reply:
I mean pizza store, where in video 2 there will be pizza store with broker showing async, then briefly in this video we can use it as intro, 

- like watch video 3 to learn what AsyncAPI is, but basically to summarize it is for describing asynchronous communication -> show pizza store
- and to understand its difference with OpenAPI you first need to know OpenAPI was designed for synchronous communication -> show pizza store what happens when you remove broker and force people to talk to each other directly, every few minutes (cook asking waiter if there is new order, every minute + waiter asking cook if order is ready every minute) - so basically complete chaos 😄 
- these are completed 2 different architectures that require different specifications and this is main difference between openapi and asyncapi, but yeah lets jump into details

---
thoughts?